data/core/logview.lua

```diff
@@ -122,12 +123,33 @@ function LogView:on_mouse_pressed(button, px, py, clicks)
 end
 
 
+function LogView:on_mouse_wheel(y, x)
+  if 0 < self.scroll.to.y then
+    if not self.user_scroll then
+      self.user_scroll = { x = self.scroll.to.x, y = self.scroll.to.y }
+    else
+      self.user_scroll.y = self.scroll.to.y
+    end
+  else
+    self.user_scroll = nil
+  end
+end
+
+
 function LogView:update()
   local item = core.log_items[#core.log_items]
   if self.last_item ~= item then
+    local lh = style.font:get_height() + style.padding.y
+    local diff_index = #core.log_items - self.last_item.index
     self.last_item = item
-    self.scroll.to.y = 0
-    self.yoffset = -(style.font:get_height() + style.padding.y)
+    self.last_item.index = #core.log_items
+    if self.user_scroll then
+      self.user_scroll.y = self.user_scroll.y + diff_index * lh
+      self.scroll.to.y = self.user_scroll.y
```

Comment:
Set `self.scroll.y` to the same value, so it doesn't animate the scroll and stays still.

Reply:
precisely what this PR is acheiving: stop autoscroll when user has scrolled

Reply:
You still need to set `self.scroll.y` (notice the absence of `to`), otherwise the "anti-scroll" will look wobbly.

Reply:
done. (just not pushed yet)